Change admin page layout UI

ferry5245 opened the pull request:
Change adminHome View
![image](https://user-images.githubusercontent.com/82063895/159846476-7e3f3842-b5b8-4bf7-b72f-fda03375f1e0.png)

Change CaasAccount View
![image](https://user-images.githubusercontent.com/82063895/159846507-d289ded8-f177-447f-b2dd-6033c911ffff.png)

Change Modal Views
![image](https://user-images.githubusercontent.com/82063895/159846561-5d072fc4-f19b-4a9e-b7d8-eab7fa5b6fbd.png)

Change EditCaasAccount View
![image](https://user-images.githubusercontent.com/82063895/159846626-7c6b5aaa-a8da-4428-9dc0-d21620377990.png)

Change delcaasconfirm View
![image](https://user-images.githubusercontent.com/82063895/159846683-2e41c71c-f712-4977-a75f-735528d85db6.png)

Change ListShift View
![image](https://user-images.githubusercontent.com/82063895/159846795-ee260577-be64-49eb-b94d-6a85ad7c7fe7.png)

Change EditShift View
![image](https://user-images.githubusercontent.com/82063895/159846830-844bd0de-3ecd-47e2-8aab-7f75748aded3.png)

Change delShiftconfirm View
![image](https://user-images.githubusercontent.com/82063895/159846864-b3ac20cc-be51-4152-98be-34922837ab9d.png)

Change Reset Plot and Reset Shift Views
![image](https://user-images.githubusercontent.com/82063895/159846936-9e829edf-d0a7-499b-90d7-8e24dd4d6af5.png)

Notes :
- ResultPlot hasn't been made yet
- Resposive page view will be made ASAP

ferry5245 committed:
Add Ferry Chandra as Contributor

ferry5245 committed:
Add composer.lock

ferry5245 committed:
Edit README.md

ferry5245 committed:
Merge branch 'Daskom-Lab-main' into ferrydev

ferry5245 committed:
Change Admin View

ferry5245 committed:
Change ResultPlot View

ferry5245 commented:
Change ResultPlot View
![image](https://user-images.githubusercontent.com/82063895/159849358-6d9325ad-017a-4267-ac8d-83a7416082a6.png)

adhiiimf commented on resources/views/adminHome.blade.php:
```diff
@@ -227,11 +229,11 @@
       </div>
       <div class="d-flex justify-content-center">
       <div class="row pt-lg-5">
```
![image](https://user-images.githubusercontent.com/66113867/159856111-c106844e-5bdb-4dd7-a4f6-27b7686f6ba1.png)
```suggestion
      <div class="row pt-lg-5 pb-lg-5">
```

adhiiimf replied:
tambahin padding bottom nya @ferry5245 , itu terlalu mepet ke bagian bawah nya, nanti coba kirim ss nya ya fer

ferry5245 replied:
Okee, aman bang, udh di commit yaaa

adhiiimf commented on resources/views/shift.blade.php:
```diff
@@ -151,10 +151,10 @@
                   </div>
                   <div >
-                    <button style="background-color:#6ec4af;font-size:17px" type="button" class="button-submit-find" data-bs-toggle="modal" data-bs-target="#delAllShift">Reset Shift</button>
+                    <button style="background-color:#8DD1C0;font-size:17px" type="button" class="button-submit-find" data-bs-toggle="modal" data-bs-target="#delAllShift">Reset Shift</button>
```
```suggestion
                    <button style="background-color:#8DD1C0;font-size:17px;color:whitesmoke;" type="button" class="button-submit-find mx-lg-2 mx-0" data-bs-toggle="modal" data-bs-target="#delAllShift">Reset Shift</button>
```

adhiiimf replied:
coba tambahin itu fer, abis nya merah sama font item nya kurang pas wkwkw sama margin x nya juga fer

ferry5245 replied:
Punten bang, setelah diikutin revisinya malah jadi begini tampilan buttonnya bang
![image](https://user-images.githubusercontent.com/82063895/159862575-6a4c0a6d-6e69-47d3-9757-f1dc41b0b070.png)

ferry5245 replied:
Solved bang, pakai width pixel jadinya

adhiiimf replied:
@ferry5245 punten fer berarti itu yang `Reset Shift` harus digelapin dikit bg colornya wkwkwk

ferry5245 replied:
Walahh, itu akhirnya diupdate lg kyknya bang, nyesuain warna merah yg lainnya, agak digelapin

adhiiimf replied:
hmm sekarang `my` buat antara bagian shift baru, reset plot, reset shift ke bagian refresh fer wkwkw wait saya coba kasih coretan

adhiiimf replied:
![image](https://user-images.githubusercontent.com/66113867/159930181-6537eecd-66e2-478d-9595-a8b6b2181758.png)
nah dikasih jarak ke bawahnya wkwk

ferry5245 replied:
Sudah bang, akhirnya pakai margin-block di CSS

adhiiimf commented on resources/views/shift.blade.php:
```diff
@@ -151,10 +151,10 @@
                     <button style="font-size:17px" type="button" class="button-submit-find" data-bs-toggle="modal" data-bs-target="#CaasInput">Shift Baru</button>
```
```suggestion
                    <button style="font-size:17px" type="button" class="button-submit-find mx-lg-2 mx-0" data-bs-toggle="modal" data-bs-target="#CaasInput">Shift Baru</button>
```

adhiiimf replied:
ini minta tolong ditambahin margin x nya fer, karena itu langsung nempel wkwkwk

adhiiimf commented on resources/views/shift.blade.php:
```diff
@@ -151,10 +151,10 @@
                   </div>
                   <div > 
-                    <button style="background-color:#fc3939;font-size:17px" type="button" class="button-submit-find" data-bs-toggle="modal" data-bs-target="#ResetPlot">Reset Plot</button>
+                    <button style="background-color:#bd3838;font-size:17px" type="button" class="button-submit-find" data-bs-toggle="modal" data-bs-target="#ResetPlot">Reset Plot</button>
```
```suggestion
                    <button style="background-color:#bd3838;font-size:17px" type="button" class="button-submit-find mx-lg-2 mx-0" data-bs-toggle="modal" data-bs-target="#ResetPlot">Reset Plot</button>
```

adhiiimf replied:
ini juga margin x nya

adhiiimf commented on resources/views/layouts/navbarAdminBack.blade.php:
```diff
@@ -1,5 +1,23 @@
+        </ul>
+        <div class="dlor-navright ">
+          <ul class="navbar-nav dlor-navright position-fixed end-0 top-0 me-lg-4 mt-lg-4" id="dlor-toggler">
+            <li class="nav-item-login">
+              <a style="font-weight: 400;" class="nav-link text-center" href="/logoutAdmin" tabindex="-1" aria-disabled="true">LOGOUT</a>
+            </li>
```
tambah `<li>` buat back ke halaman utama admin fer

adhiiimf replied:
tapi bebas, kalau mau di `PR` selanjutnya untuk ini gapapa, tapi takut lupa aja wkwkwk

ferry5245 replied:
Udah dibuat bang
![image](https://user-images.githubusercontent.com/82063895/159878082-b0d99dc0-62a1-40f6-8813-195a21efe987.png)

adhiiimf commented on resources/views/home.blade.php:
```diff
@@ -58,11 +60,116 @@
+                <div class="Welcome-text-title">Selamat Datang</div>
+                <div class="Welcome-text-title">Calon Asisten Daskom</div>
+                <div class="Welcome-text-title">2022</div>
+                <!-- <span class="Welcome-text-title">
+                  <p>Selamat Datang</p>
+                  <p>Calon Asisten Daskom</p> 
+                  <p>2022</p> 
+                  </span> -->
```
```suggestion
      
```
hapus aja ini wkwk nanti tinggal liat history git nya kalau emang dibutuhin lagi

adhiiimf commented on public/css/list.css:
```diff
@@ -1,3 +1,3 @@
-#list-section{
+/* #list-section{
```
hapus aja ini fer wkwk

adhiiimf reviewed (changes requested):
### NICE Pull Request
Mantapp @ferry5245, dilihat dulu ya komen dan saran dari sayaa, nanti coba diss juga hasil perubahannya, kalau udah oke, langsung commit terus push lagi, abis itu re-request review dari saya ya fer, keren euyy wkwkwk

fakhrip commented on README.md:
```diff
@@ -15,7 +15,11 @@ Daskom recruitment website is a website related to the recruitment of new [Dasko
 - [Dini Rizqi Amalia](https://github.com/dinudinni) [DIN]
-- [Fahira Qaulifa](https://github.com/fahiraq) [FAQ]
\ No newline at end of file
+- [Fahira Qaulifa](https://github.com/fahiraq) [FAQ]
+>>>>>>> a91840f7ef106a42cdb0752bac0bdc34ddec0ea6
```
ini salah fer, cara yang bener buat nge fix conflict itu contohnya begini (sekalian aja itu diganti jadi gini ya)
```suggestion
- [Fahira Qaulifa](https://github.com/fahiraq) [FAQ]
```

(ini ilangin bagian `>>>>>` karena itu cuman buat ngasih tau yang conflict doang dari git nya)

ferry5245 replied:
Oalahhh, tapi itu udh ke solve bang, jadi aman kah???
Makasih insightnya bang XD

fakhrip replied:
udah kamu solve ? kalau udah yaudah bagus fer wkwk, sipp sama sama, ntar tak review lagi ya kalau dah semua nya di resolve, habis adhi kek biasa

fakhrip replied:
biar clear, maksud solve disini tuh, kamu ubah kek yang saya saranin ya wkwk

ferry5245 replied:
Udah bang, ternyata ga ke commit wkwkwkwk

fakhrip commented on README.md:
```diff
@@ -15,7 +15,11 @@ Daskom recruitment website is a website related to the recruitment of new [Dasko
 
 ## Develop by:
 - [Adhi Muhammad Fahmi](https://github.com/adhiiimf) [AMF]
+<<<<<<< HEAD
+- [Ferry Chandra](https://github.com/ferry5245) [FEY]
+=======
```
ini salah fer, cara yang bener buat nge fix conflict itu contohnya begini (sekalian aja itu diganti jadi gini ya)
```suggestion
```

(dihapus karena nama kamu udah ada di bawah nya)

ferry5245 replied:
Oke bang, siap

ferry5245 committed:
Add Space Under the Main Content

ferry5245 committed:
Update resources/views/adminHome.blade.php

Add Bottom Padding to Main View

Co-authored-by: Adhi Muhammad Fahmi <66113867+adhiiimf@users.noreply.github.com>

fakhrip commented on resources/views/home.blade.php:
```diff
@@ -58,11 +60,116 @@
+                  </a>
+                </div>
+              </div>
+              @if($plotactive->isPlotActive==1)
```
karena ini php jadi kalau buat comparison selalu biasain pake `===` ya fer, biar aman (biar ga kena celah keamanan namanya type juggling)
```suggestion
              @if($plotactive->isPlotActive===1)
```

ferry5245 replied:
Ini utk semua if else kah bang?

fakhrip replied:
yoii bener fer

fakhrip commented on resources/views/home.blade.php:
```diff
@@ -58,11 +60,116 @@
+                  </a>
+                </div>
+              </div>
+              @elseif($firstmeet->isPlotFirstmeet==1 && $plotactive->isPlotActive==NULL && $caas->urut_tahap==1)
```
```suggestion
              @elseif($firstmeet->isPlotFirstmeet===1 && $plotactive->isPlotActive===NULL && $caas->urut_tahap===1)
```

fakhrip commented on resources/views/layouts/navbarAdminBack.blade.php:
```diff
@@ -1,5 +1,23 @@
+        </div>
+    </nav>
+  </section>
+<!-- <section id="nav-section">
```
ini comment comment jangan lupa hapus hapusin dulu biar ga bingung kedepannya

ferry5245 replied:
Okeyy, noted bang

fakhrip commented on resources/views/shift.blade.php:
```diff
@@ -22,7 +22,7 @@
 </head>
-<body id="list-section">
+<body style="background-color: #2c4d16" id="list-section">
 <!-- Modal Caas Input -->
 <!-- Button trigger modal -->
   <!-- Modal -->
```
ini hapusjuga

ferry5245 replied:
Siap bang

fakhrip commented on resources/views/shift.blade.php:
```diff
@@ -169,24 +169,24 @@
-                        <th>Waktu</th>
-                        <th>Kuota</th>
-                        <th>OPSI</th>
+                        <th style="color:black;font-weight:700;background-color:#f1b442" class="mobile-hide">No.</th>
+                        <th style="color:black;font-weight:700;background-color:#f1b442" class="mobile-hide">SHIFT</th>
+                        <th style="color:black;font-weight:700;background-color:#f1b442">Tanggal</th>
+                        <th style="color:black;font-weight:700;background-color:#f1b442">Waktu</th>
+                        <th style="color:black;font-weight:700;background-color:#f1b442">Kuota</th>
+                        <th style="color:black;font-weight:700;background-color:#f1b442">OPSI</th>
```
ini kan style nya sama untuk beberapa elemen, jadi kamu buat class aja mending terus styling di class nya fer, biar ga repetitive dan panjang begini

ferry5245 replied:
Awalnya berniat begitu bang, tapi ternyata stylenya malah ketimpa sama file table.scss, mungkin dari Framework kah?

ferry5245 replied:
Oke aman bang, ID works well, untung keinget Specificity wkwkwkwk

fakhrip replied:
wkwkw sipp mantap mantap

fakhrip reviewed (changes requested):
Sipp mantap dah bagus PR nya, buat problem design ikutin arahan dari adhi aja ya, sisanya ini saya nambahin dikit dikit doang...

ferry5245 committed:
- Remove unnecessary comments from NavbarAdmin
- Remove unnecessary comments from adminHome
- Shift Update View
- Remove unnecessary comments from Home
- Add Navbar to DelShift
- Edit resultplot Styles
- EditCaasAccount (Add Navbar, Remove Unecessary Comment)
- Add Navbar to delcaas
- CaasAccount (Remove Comment, Edit Styles)
- Edit form-style.css
- Delete list.css File
- Edit README.md

ferry5245 committed:
Merge branch 'ferrydev' of https://github.com/ferry5245/daskomrec22 into ferrydev

ferry5245 committed:
Merge branch 'main' into ferrydev

ferry5245 committed:
Merge branch 'ferry5245-ferrydev'

adhiiimf commented:
@ferry5245 boleh minta ssan hasil editan terbaru nya fer?

ferry5245 commented:
Edit Shift View
![image](https://user-images.githubusercontent.com/82063895/159933277-484f14bf-005a-4a66-a43c-46bdfd187ba2.png)

Add Navbar delShiftconfirm
![image](https://user-images.githubusercontent.com/82063895/159933399-7f310846-28ea-479e-b1b4-ba27901b0b92.png)

EditCaasAccount
![image](https://user-images.githubusercontent.com/82063895/159933634-d9750b79-06c3-49bf-ad37-d0ddc03db468.png)

adhiiimf commented:
> Edit Shift View ![image](https://user-images.githubusercontent.com/82063895/159933277-484f14bf-005a-4a66-a43c-46bdfd187ba2.png)
> 
> Add Navbar delShiftconfirm ![image](https://user-images.githubusercontent.com/82063895/159933399-7f310846-28ea-479e-b1b4-ba27901b0b92.png)
> 
> EditCaasAccount ![image](https://user-images.githubusercontent.com/82063895/159933634-d9750b79-06c3-49bf-ad37-d0ddc03db468.png)

![image](https://user-images.githubusercontent.com/66113867/159936293-e209f58c-d84b-4348-8a4b-e941203bf60f.png)

itu ganti text color nya jadi warna gelap aja fer

adhiiimf commented on resources/views/layouts/navbarCaasLogout.blade.php:
```diff
@@ -1,5 +1,23 @@
+        </div>
+    </nav>
+  </section>
+<!-- <section id="nav-section">
     <nav class="navbar navbar-expand-lg dlor-navbar">
         <div class="container-fluid">
           <a class="navbar-brand" href=""><img src="{{asset('/assets/dlor.png')}}" alt="logo" class="dlor-logonav"></a>
```
hapus aja ini bagian yg dikomenin nya

adhiiimf commented on resources/views/layouts/navbarCaasLogout.blade.php:
```diff
@@ -12,4 +30,4 @@
       </nav>
-</section>
\ No newline at end of file
+</section> -->
```
ini juga hapus, berarti dari line 20-33

adhiiimf commented on resources/views/layouts/navbarAdminBack.blade.php:
```diff
@@ -1,15 +1,22 @@
+        </ul>
+        <div class="dlor-navright ">
+          <ul class="navbar-nav dlor-navright position-fixed end-0 top-0 me-lg-4 mt-lg-4" id="dlor-toggler">
+            <li class="nav-item-login mx-3">
+              <a style="font-weight: 400;" class="nav-link text-center" href="/adminHome" tabindex="-1" aria-disabled="true">HOME</a>
+            </li>
+            <li class="nav-item-login mx-3">
+              <a style="font-weight: 400;" class="nav-link text-center" href="/logoutAdmin" tabindex="-1" aria-disabled="true">LOGOUT</a>
+            </li>
```
```suggestion
            <li class="nav-item-login mx-2">
              <a style="font-weight: 400;" class="nav-link text-center" href="/adminHome" tabindex="-1" aria-disabled="true">HOME</a>
            </li>
            <li class="nav-item-login mx-2">
              <a style="font-weight: 400;" class="nav-link text-center" href="/logoutAdmin" tabindex="-1" aria-disabled="true">LOGOUT</a>
            </li>
```
jadiin mx-2 aja biar gaterlalu jauh juga, sama mau nanya, ini udah disiapin buat mobile nya? atau emang baru buat desktop aja?

adhiiimf commented on resources/views/resultplot.blade.php:
```diff
@@ -79,25 +69,25 @@
-                        <th>Waktu</th>
-                        <th  class="mobile-hide">Kuota</th>
-                        <th>LIST</th>
+                        <th id="table-backgrounds" class="table-content-custom mobile-hide">No.</th>
+                        <th id="table-backgrounds" class="table-content-custom mobile-hide">SHIFT</th>
+                        <th id="table-backgrounds" class="table-content-custom">Tanggal</th>
+                        <th id="table-backgrounds" class="table-content-custom">Waktu</th>
+                        <th id="table-backgrounds" class="table-content-custom mobile-hide">Kuota</th>
+                        <th id="table-backgrounds" class="table-content-custom">LIST</th>
```
```suggestion
                        <th class="table-content-custom mobile-hide">No.</th>
                        <th class="table-content-custom mobile-hide">SHIFT</th>
                        <th class="table-content-custom">Tanggal</th>
                        <th class="table-content-custom">Waktu</th>
                        <th class="table-content-custom mobile-hide">Kuota</th>
                        <th class="table-content-custom">LIST</th>
```

styling yang ada di id tadi satuin aja ke `table-content-custom`

ferry5245 replied:
gabisa bang, untuk background nya ketimpa sm file lain, makanya itu juga udah dibuat alternatifnya

adhiiimf commented on resources/views/resultplot.blade.php:
```diff
@@ -79,25 +69,25 @@
-                        <td>{{ \Carbon\Carbon::parse($p->hari)->isoFormat('dddd, D MMMM Y') }}</td>
-                        <td>{{ $p->jam_start }} - {{ $p->jam_end }} WIB</td>
-                        <td  class="mobile-hide">{{ $p->kuota }}</td>
+                        <td id="table-backgrounds" class="table-content-custom mobile-hide">{{ $no++ }}</td>
+                        <td id="table-backgrounds" class="table-content-custom mobile-hide">{{ $p->namashift }}</td>
+                        <td id="table-backgrounds" class="table-content-custom">{{ \Carbon\Carbon::parse($p->hari)->isoFormat('dddd, D MMMM Y') }}</td>
+                        <td id="table-backgrounds" class="table-content-custom">{{ $p->jam_start }} - {{ $p->jam_end }} WIB</td>
+                        <td id="table-backgrounds" class="table-content-custom mobile-hide">{{ $p->kuota }}</td>
```
```suggestion
                        <td class="table-content-custom mobile-hide">{{ $no++ }}</td>
                        <td class="table-content-custom mobile-hide">{{ $p->namashift }}</td>
                        <td class="table-content-custom">{{ \Carbon\Carbon::parse($p->hari)->isoFormat('dddd, D MMMM Y') }}</td>
                        <td class="table-content-custom">{{ $p->jam_start }} - {{ $p->jam_end }} WIB</td>
                        <td class="table-content-custom mobile-hide">{{ $p->kuota }}</td>
```
ini juga

adhiiimf commented on public/css/form-style.css:
```diff
@@ -152,42 +150,41 @@
     transition: 0.5s;
 }
 
+.table-content-custom{
+    color:black;
+    font-weight:700;
+}
+
+#table-backgrounds{
+    background-color:#f1b442;
+}
+
```
```suggestion
.table-content-custom{
    color: black;
    font-weight: 700;
    background-color: #f1b442!important;
}
```

satuin aja

adhiiimf replied:
@ferry5245 pakai !important coba ya fer

adhiiimf commented on public/css/home.css:
```diff
@@ -1,40 +1,55 @@
-    padding: 0px;
-    width: 210px;
-    border-radius: 25px;
+    /* display: inline; */
```
ini hapus aja tinggal liat history wkwkw
```suggestion
```

adhiiimf commented on public/css/login-caas.css:
```diff
@@ -0,0 +1,119 @@
+        margin: 2vw;
+    }
+    .text-nim-head {
+        color: #f9d098;
```
```suggestion
```

hapus aja karena sama kyk yang di default nya kan? di line 21

adhiiimf commented on public/css/login-caas.css:
```diff
@@ -0,0 +1,119 @@
+        font-size: 10vw;
+    }
+    .form-style {
+        border-radius: 15px;
+        font-size: 30px;
```
```suggestion
```
hapus juga karena di default atas nya sama

adhiiimf commented on public/css/login-caas.css:
```diff
@@ -0,0 +1,119 @@
+    }
+    .form-style {
+        border-radius: 10px;
+        font-size: 30px;
```
```suggestion
```
ini juga hapus aja fer kyk komen sebelumnya

adhiiimf commented on resources/views/CaasAccount.blade.php:
```diff
@@ -139,29 +138,29 @@
-                        <th>Status</th>
-                        <th>Tahap</th>
-                        <th>OPSI</th>
+                        <th style="color:black;font-weight:700;background-color:#f1b442" class="mobile-hide">No.</th>
+                        <th style="color:black;font-weight:700;background-color:#f1b442" class="mobile-hide">Nama</th>
+                        <th style="color:black;font-weight:700;background-color:#f1b442">NIM</th>
+                        <th style="color:black;font-weight:700;background-color:#f1b442" class="mobile-hide">Email</th>
+                        <th style="color:black;font-weight:700;background-color:#f1b442">Status</th>
+                        <th style="color:black;font-weight:700;background-color:#f1b442">Tahap</th>
+                        <th style="color:black;font-weight:700;background-color:#f1b442">OPSI</th>
```
Ini komenan aja, mungkin buat kedepannya dibuat class aja di css nya, biar rapih, kalau kyk gini repot ganti-ganti nya wkwkw

adhiiimf commented on resources/views/CaasAccount.blade.php:
```diff
@@ -139,29 +138,29 @@
                     <tr>
-                      <td class="mobile-hide">{{ $no++ }}</td>
-                        <td class="uppercase mobile-hide">{{ $p->nama }}</td>
+                      <td style="color:black;font-weight:700;background-color:#f1b442" class="mobile-hide">{{ $no++ }}</td>
+                        <td style="color:black;font-weight:700;background-color:#f1b442" class="uppercase mobile-hide">{{ $p->nama }}</td>
```
ini juga

adhiiimf commented on resources/views/CaasAccount.blade.php:
```diff
@@ -139,29 +138,29 @@
+                        <td style="color:black;font-weight:700;background-color:#f1b442" class="uppercase mobile-hide">{{ $p->nama }}</td>
                         <td style="color:black;font-weight:700;background-color:#f1b442">{{ $p->nim }}</td>
-                        <td class="mobile-hide">{{ $p->email }}</td>
+                        <td style="color:black;font-weight:700;background-color:#f1b442" class="mobile-hide">{{ $p->email }}</td>
```
ini juga

adhiiimf replied:
dan kawan-kawan nya yg kurang lebih seperti itu wkwk

adhiiimf commented on resources/views/adminHome.blade.php:
```diff
@@ -22,7 +22,7 @@
     
 </head>
-<body>
+<body style="background-color: #2c4d16">
```
ini juga, next PR dibuat jadi class aja

adhiiimf commented on resources/views/adminHome.blade.php:
```diff
@@ -226,12 +215,12 @@
                 <a style="text-decoration: none" href="/CaasAccount">
-                <button style="background-color: #404040;color: whitesmoke;" class="home-button">
-                  <div class="menu-box-home">
+                <button style="background-color: #404040;color: #f9d098;" class="home-button-admin">
```
ya ini juga wkwk dan yg sejenis nya kayak gini nanti dijadiin class aja

adhiiimf reviewed (changes requested):
Oke udah direview lagi ya fer, sori kalau ada permintaan atau saran request change lagi wkwk biar sekalian ajaa, semangatt, udah keren iniii 💯👌👍😎

adhiiimf reviewed (changes requested):
Update review buat case css for table nya fer

ferry5245 committed:
- Edit Class for resultplot, shift
- Edit form-style.css

ferry5245 committed:
Update public/css/form-style.css

Co-authored-by: Adhi Muhammad Fahmi <66113867+adhiiimf@users.noreply.github.com>

ferry5245 committed:
Update public/css/form-style.css

Co-authored-by: Adhi Muhammad Fahmi <66113867+adhiiimf@users.noreply.github.com>

ferry5245 committed:
- Add responsive view for Admin
- Style cleanup

ferry5245 committed:
Edit form-style.css

ferry5245 committed:
Add comment for media query

ferry5245 commented:
Mobile View adminHome
![image](https://user-images.githubusercontent.com/82063895/159972618-9c90f7d8-a85c-4175-bd2c-e0cdbca8081b.png)

Mobile View CaasAccount
![image](https://user-images.githubusercontent.com/82063895/159972696-8edbbdcb-2572-41d3-b949-ad2ddd9acb69.png)

Mobile View EditCaasAccount
![image](https://user-images.githubusercontent.com/82063895/159972793-0fe08a59-98ff-4859-b37c-c8092fd0a088.png)

Mobile View delcaasconfirm
![image](https://user-images.githubusercontent.com/82063895/159972873-a593772c-1ce0-4400-b54b-581ed780e6e2.png)

Mobile View ListShift
![image](https://user-images.githubusercontent.com/82063895/159972984-53f708c1-fa13-4b80-ac1c-4b7e36d349e0.png)

Mobile View Modal
![image](https://user-images.githubusercontent.com/82063895/159973044-bfc7b489-9e46-4844-ba07-a51ef2fa7c36.png)

Mobile View EditShift
![image](https://user-images.githubusercontent.com/82063895/159973187-3e74e7a2-c427-4106-b737-6de46e99238e.png)

Mobile View delShiftconfirm
![image](https://user-images.githubusercontent.com/82063895/159973317-eb6074bd-8819-4db2-a7cc-cd53382df336.png)

adhiiimf reviewed (approved):
Finally fer sejauh ini aman, saya approve, nanti kalau ditengah ternyata ada yang kurang mungkin saya bikin issue biar didevelop wkwkwk, mantap ferr

fakhrip reviewed (approved):
Sip saya approve kalau dari adhi dah resolved semua, mantapp fer lanjutkan :+1: , boleh di merge sok btw
